src/app/shared/select/select-item/select-item.component.ts

```diff
@@ -46,11 +46,21 @@ export class SelectItemComponent {
   @Input()
   public disabled: boolean;
 
+  @Input()
+  public removable: boolean = false;
+
   @Output()
   public select = new EventEmitter<any>();
 
+  @Output()
+  public removed = new EventEmitter();
```

Comment:
Libraries usually distinguish between `remove` and `removed` events. The former means that the remove action has been triggered while the latter means that the action has successfully completed.

But this event here is about triggering the remove action, right? If so, it would be better to name it `remove` in order to avoid confusion.